python/pyxir/contrib/target/components/DPUCAHX8L/u280_op_support.py

```diff
@@ -0,0 +1,223 @@
+    """ Check whether we can execute the provided Padding operator
+        on the DPUCAHX8L-u280 target """
+    return is_padding_supported(X, bXs, tXs)
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "Pooling")
+def pooling_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """ Check whether we can execute the provided Pooling operator
+        on the DPUCAHX8L-u280 target """
+    return is_pooling_supported(
+        X=X,
+        bXs=bXs,
+        tXs=tXs,
+        channel_parallel=32,
+        bank_depth=4096,
+        max_pool_min_kernel=2,
+        max_pool_max_kernel=8,
+        max_pool_min_stride=1,
+        max_pool_max_stride=8,
+        avg_pool_min_kernel=1,
+        avg_pool_max_kernel=8,
+        avg_pool_min_stride=1,
+        avg_pool_max_stride=8,
+        max_pool_kernel_valid=[ 2, 3, 5, 7, 8],
+        avg_pool_kernel_valid=[2, 3, 5, 7, 8],
+    )
+
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "Mean")
+def mean_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """ Check whether we can execute the provided Mean operator
+        on the DPUCAHX8L-u280 target """
+    axes = X.attrs["axes"]
+    keepdims = X.attrs["keepdims"]
+    return len(axes) == 2 and keepdims
+
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "LeakyReLU")
+def leaky_relu_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """ Check whether we can execute the provided LeakyRelu operator
+        on the DPUCAHX8L-u280 target """
+    assert len(bXs) == 1
+    bX = bXs[0]
+    alpha = X.attrs["alpha"]
+    # LeakyRelu not supported after depthwise conv2d
+    return math.isclose(alpha, 0.1, rel_tol=1e-5) and bX.type[0] in [
+        "Convolution",
+        "Conv2DTranspose",
+        "BatchNorm",
+        "Scale",
+    ]
+
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "pReLU")
+def prelu_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """ Check whether we can execute the provided pRelu operator
+        on the DPUCAHX8L-u280 target """
+    # Only LeakyReLU: alpha == 0.1 supported
+    assert len(bXs) == 1
+    bX = bXs[0]
+    alpha = X.attrs["alpha"]
+    # LeakyRelu not supported after depthwise conv2d
+    return math.isclose(alpha, 0.1, rel_tol=1e-5) and bX.type[0] in [
+        "Convolution",
+        "Conv2DTranspose",
+        "BatchNorm",
+        "Scale",
+    ]
+
+
+@pyxir.register_op_support_check('DPUCAHX8L-u280', 'ReLU')
+def relu6_op_support(X, bXs, tXs):
+    # Type: (XLayer, List[XLayer], List[XLayer]) -> boolean
+    """ Check whether we can execute the provided ReLU operator
+        on the zcu104 target """
+
+    # TODO always?
+
+    return True
+
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "ReLU6")
+def relu6_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """ Check whether we can execute the provided ReLU6 operator
+        on the DPUCAHX8L-u280 target """
+    return False
```

Comment:
Relu6 is supported for DPU after Convolution, Transposed Conv2D, BatchNorm, Scale, Eltwise

Reply:
added